src/main/scala/beam/agentsim/agents/household/FastHouseholdCAVScheduling.scala

```diff
@@ -297,11 +297,16 @@ case class CAVSchedule(schedule: List[MobilityRequest], cav: BeamVehicle, occupa
               newMobilityRequests = newMobilityRequests :+ orig.copy(routingRequestId = Some(embodyReq.requestId))
               Some(RouteOrEmbodyRequest(None, Some(embodyReq)))
             case None =>
+              val personId = orig.person match {
```

Comment:
nitpick: you can use `orig.person.map(_.personId)`

Reply:
replaced code as suggested